org.garudalinux.firedragon.yml

```diff
@@ -0,0 +1,172 @@
+        sha256: ee8f3ef946156ad3406fdf45feedbdcd932dbd211ab4f16f75eba4f36fb2f6c0
+  - name: sound-theme-freedesktop
+    sources:
+      - type: git
+        url: https://salsa.debian.org/gnome-team/sound-theme-freedesktop.git
+        tag: upstream/0.8
+  - name: pcsc-lite
+    config-opts:
+      - --disable-libudev
+      - --disable-libsystemd
+      - --without-systemdsystemunitdir
+      - --disable-serial
+      - --disable-usb
+      - --disable-documentation
+      - --disable-polkit
+    post-install:
+      - rm /app/sbin/pcscd
+      - rmdir /app/sbin || true
+    sources:
+      - type: archive
+        url: https://salsa.debian.org/rousseau/PCSC/-/archive/2.0.3/PCSC-2.0.3.tar.bz2
+        sha256: 35ce63ff9f8fa34b67fbc791be08b3daba621263545e3d0fe1e480301f10a41d
+  - name: dotconf
+    cleanup:
+      - "*"
+    sources:
+      - type: archive
+        url: https://github.com/williamh/dotconf/archive/refs/tags/v1.3.tar.gz
+        sha256: 7f1ecf40de1ad002a065a321582ed34f8c14242309c3547ad59710ae3c805653
+      - type: script
+        commands:
+          - autoreconf -fiv
+        dest-filename: autogen.sh
+  - name: libspeechd
+    config-opts:
+      - --disable-python
+      - --with-systemdsystemunitdir=''
+      - --disable-rpath
+      - --disable-static
+      - --with-kali=no
+      - --with-baratinoo=no
+      - --with-ibmtts=no
+      - --with-voxin=no
+      - --without-oss
+    sources:
+      - type: archive
+        url: https://github.com/brailcom/speechd/releases/download/0.11.5/speech-dispatcher-0.11.5.tar.gz
+        sha256: 1ce4759ffabbaf1aeb433a5ec0739be0676e9bdfbae9444a7b3be1b2af3ec12b
+    cleanup:
+      - /bin
+      - /etc
+      - /libexec
+      - /lib/speech-dispatcher
+      - /share/sounds/speech-dispatcher
+      - /share/speech-dispatcher
+  - name: firedragon
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 org.garudalinux.firedragon.appdata.xml -t /app/share/metainfo/
+      - mkdir -p /app/lib/ffmpeg
+      - mv firedragon-*.tar.bz2 firedragon.tar.bz2
+      - tar -xvf ./firedragon.tar.bz2 -C /app/lib
+      - cp -r ./src/* /app/
+      - chmod +x /app/bin/firedragon
+      - >
+        for size in 16 32 48 64 128; do
+          mkdir -p "/app/share/icons/hicolor/${size}x${size}/apps"
+          cp "/app/lib/firedragon/browser/chrome/icons/default/default${size}.png" "/app/share/icons/hicolor/${size}x${size}/apps/org.garudalinux.firedragon.png"
+        done
+      - >
+        export DISTRIBUTION_VERSION=$(cat org.garudalinux.firedragon.appdata.xml | sed -r 's/\s*?<release\sversion="([0-9\.\-]*?)"\sdate="([0-9\-]*?)".*?\/>\s*?/\1_\2/gp;d' | head -1)
+
+        echo $DISTRIBUTION_VERSION | grep -E "^[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?_[0-9]{4}-[0-9]{2}-[0-9]{2}$" # validation
+
+        echo "pref(\"distribution.version\", \"$DISTRIBUTION_VERSION\");" >> /app/lib/firedragon/defaults/pref/flatpak-prefs.js
+    sources:
+      - type: file
```

Comment:
```suggestion
      - type: archive
```

Reply:
Fixed in 971cf4e61c45647b718ba80152c4ad517f2c20b1